src/extensions/prometheus-metrics/prometheus_exporter_test.go

```diff
@@ -14,11 +15,11 @@ func TestExporter(t *testing.T) {
 
 	exporter := NewExporter(metricReport1)
 
-	assert.Equal(t, metricReport1, exporter.GetLatestMetricReport())
+	assert.Equal(t, metricReport1, exporter.GetLatestMetricReports()[0])
 
-	exporter.SetLatestMetricReport(metricReport2)
+	exporter.SetLatestMetricReport(&metrics.MetricsReportBundle{Data: []*proto.MetricsReport{metricReport2}})
```

Comment:
Can you update the unit test to have more than one report in the MetricsReportBundle?

Reply:
will do